src/Workspaces/Core/Portable/Workspace/Workspace.cs

```diff
@@ -250,10 +250,6 @@ private protected (Solution oldSolution, Solution newSolution) SetCurrentSolutio
                     var newSolution = data.transformation(oldSolution);
 
                     // Attempt to unify the syntax trees in the new solution (unless the option is set disabling that).
```

Comment:
```suggestion
```

The parenthetical in the comment doesn't make sense anymore.

Reply:
will fixup comment in followup.